apps/src/templates/teacherDashboard/DialogFooter.jsx

```diff
@@ -29,16 +32,20 @@ const style = {
  */
 export default class DialogFooter extends Component {
   static propTypes = {
+    rightAlign: PropTypes.bool,
     children: PropTypes.any
   };
 
   render() {
+    let buttonRowStyle = this.props.rightAlign
+      ? {...style.buttonRow, ...style.rightAlign}
```

Comment:
I'm curious if the multiple flex attributes will cause issues in IE. I'm happy to test this on my machine if you'd like.

Reply:
I checked on SauceLabs and it behaves as expected on IE11Win10.